packages/@aws-cdk/aws-apigatewayv2-integrations/lib/websocket/lambda.ts

```diff
@@ -36,9 +36,16 @@ export class LambdaWebSocketIntegration implements IWebSocketRouteIntegration {
       }),
     });
 
+    const integrationUri = Stack.of(route).formatArn({
```

Comment:
This URI is used in quite a few places. 

This - https://github.com/aws/aws-cdk/pull/13181/files#diff-2c8f1a771d6f62decf7e5ef20ebf6de6218c25ade0f1531ed5e78609f3bc23bdR110 - being another instance.

Let's move this as a getter - `invocationUri` - on `IFunction` and reuse?

Reply:
This seems very specific to `apigateway` service and is evident from the `invocationUri` itself. I am not sure if it would be a correct place to put this in `IFunction`. If this use case is common for any other service as well, then we can add a helper in `IFuntion` which accepts `service` as an argument. Thoughts?